app/portainer/views/endpoints/edit/endpointController.js

```diff
@@ -275,6 +304,10 @@ function EndpointController(
         // Check if the environment is docker standalone, to decide whether to show the GPU insights box
         const isDockerEnvironment = endpoint.Type === PortainerEndpointTypes.DockerEnvironment;
         if (isDockerEnvironment) {
+          if (endpoint.URL.indexOf('tcp://') === 0) {
```

Comment:
```suggestion
          if (endpoint.URL.startsWith('tcp://')) {
```

Reply:
maybe move it to `app/react/portainer/environments/utils/index.ts`?

Reply:
fixed